wire-ios-system/Source/Logging/WireLogger.swift

```diff
@@ -188,3 +188,13 @@ public extension WireLogger {
     static let system = WireLogger(tag: "system")
     static let featureConfigs = WireLogger(tag: "feature-configurations")
 }
+
+/// Class to proxy WireLogger methods to Objective-C
+class WireLoggerObjc: NSObject {
+
+    @objc
+    static func assertionDumpLog(_ message: String) {
```

Comment:
Should we add `@objcMembers` on the whole class, since it's called `WireLoggerObjc´?